### ISSUE TITLE
Add --enable for Safari 11

### ISSUE THREAD

**Issue by gauntface** [GoogleChromeLabs/selenium-assistant#100]

**Comment by gauntface** [comment]
This is the driver than needs this:

```
$ safaridriver
Could not start server: must specify at least one configuration argument.

Usage: safaridriver [options]
	-h, --help                Prints out this usage information.
	-p, --port                Port number the driver should use. If the server
	                          is already running, the port cannot be changed.
	                          If port 0 is specified, a default port will be used.
	--enable                  Applies configuration changes so that subsequent WebDriver
	                          sessions will run without further authentication.
```

**Comment by jeffposnick** [comment]
This should be fixed by #120.
